content/people/hannah-anderson.md

```diff
@@ -0,0 +1,13 @@
+---
+avatar: /images/people/hannah-anderson.jpg?v=0
```

Comment:
Not sure if this will break an image

Reply:
Good Q, thanks. Really just an artifact of bootstrappin' off fireside.